scripts/cpbusylib

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
```

Comment:
This file is not needed at all because if busybox is not statically linked, the libraries should be copied over by the build script--mkiinitramfs-ll.$SHELL. If not, there is a bug which shall be fixed. Just take a look at the script where busybox copy is handled to see this.